src/main/docker/Dockerfile

```diff
@@ -1,6 +0,0 @@
-FROM java:7
-VOLUME /tmp
-ADD eureka-0.0.1-SNAPSHOT.jar /app.jar
-RUN bash -c 'touch /app.jar'
-EXPOSE 8761
-ENTRYPOINT ["java","-Djava.security.egd=file:/dev/./urandom","-jar","/app.jar"]
```

Comment:
You've missed part of the entrypoint

Reply:
I did not.

The touchy its useless. The volume can be made outside. And the random thing its to address a bug already solved.

Reply:
Explaining those kinds of things upfront in a PR is helpful. Where is the 'random thing' solved?

Reply:
its resolved here: https://bugs.java.com/view_bug.do?bug_id=6202721

Reply:
https://wiki.apache.org/tomcat/HowTo/FasterStartUp#Entropy_Source

Reply:
it even mentions the bug you linked to. It solved the funny `/./` syntax, not the shortage of entropy.

Reply:
Look like over best pratices. Maybe let that on java opts env options?

Reply:
the random entropy jar option needs to stay, otherwise, I'll go ahead and close this.

Reply:
Ok. Close. I will make another pr. Tx